src/formatDate.js

```diff
@@ -9,6 +9,64 @@
  */
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const dateSeparated = date.split(fromFormat[3]);
+  let day = '';
+  let month = '';
+  let year = '';
+
+  for (let i = 0; i < 3; i++) {
+    switch (fromFormat[i]) {
+      case 'YYYY':
+        year = dateSeparated[i];
+        break;
+      case 'YY':
+        year = dateSeparated[i];
```

Comment:
The case for 'YY' is redundant as it performs the same action as the case for 'YYYY'. You can combine both 'YYYY' and 'YY' cases to simplify the code.